kratos/utilities/delaunator_utilities.h

```diff
@@ -0,0 +1,73 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_DELAUNATOR_UTILITIES)
+#define KRATOS_DELAUNATOR_UTILITIES
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+/**
+ * @namespace DelaunatorUtilities
+ * @ingroup KratosCore
+ * @brief This namespace includes several utilities using the library delaunator-cpp
```

Comment:
@loumalouomega here is still delaunator

Reply:
But it is not mandatory, only if included, do you want me to absolutely remove it?